### ISSUE TITLE
DeterminsticTravelTimes: verification failed

### ISSUE THREAD

**Issue by luchengqi7** [matsim-org/alonso-mora#12]
Hi Sebastian,

If I turn on the Deterministic travel time check (Alonso-mora config group: setCheckDeterminsticTravelTimes(true)), then the check will fail even under a network with very large capacity (e.g., capacity factors = 100000). Do you also experience similar issue? 

I think we have similar problems in the standard DRT contrib. A similar consistency check of the vehicle schedule and insertion plan will also lead to the error. And therefore the check was temporarily disabled.

Best regards,
Chengqi Lu

**Comment by sebhoerl** [comment]
Hi Chengqi, this assumes that capacities are close to infinity, but also that travel times don't vary over the day. The latter is the case for you, isn't it?

**Comment by luchengqi7** [comment]
Yes, as the starting point, I am trying the algorithm on a deterministic and time-invariant network. Since the fleet is not so large in the scenario, so I think the travel time will remain constant throughout the day. Therefore, I am not sure why the test is failing.

**Comment by sebhoerl** [comment]
Normally, if capacities are set to 9999999 (flow and storage) and times are constant over the day, it should work.

**Comment by sebhoerl** [comment]
Tarek had the same issue, need to check what is special about the NY use case / the cases in the unit tests that make this work. Probably some config option
